src/queue.ts

```diff
@@ -25,11 +25,19 @@ export class Queue<TItem> extends EventEmitter {
     this.options = options
   }
 
-  async push (item : TItem) : Promise<void> {
-    await this.options.sqs.sendMessage({
+  async push (item : TItem, messageGroupId?: string) : Promise<void> {
```

Comment:
I think the second parameter should be an object, so it's more easily extendable in the future.

Reply:
Fixed